wrapper/src/vmaf.cpp

```diff
@@ -922,13 +925,13 @@ void BootstrapVmafQualityRunner::_set_prediction_result(
     size_t num_models = predictionStructs.at(0).vmafMultiModelPrediction.size();
     std::vector<double> perModelScore;
     // character array to put the name of the vmaf bootstrap model, e.g. vmaf_0001 is the first one
-    char char_buffer[50];
+    char char_buffer[BOOTSTRAP_MODEL_NAME_BUF_SIZE];
```

Comment:
Suggestion: use c++ style std::string instead of c style char. For example:
https://github.com/Netflix/vmaf/blob/master/wrapper/src/vmaf.cpp#L147

Reply:
just updated the PR with a commit, what do you think?

Reply:
Looks good. Go ahead with the merge!

Reply:
done, thanks!